libs/unity/library/Runtime/Scripts/NativeRender/NativeVideo.cs

```diff
@@ -36,7 +46,9 @@ internal class NativeVideo : IDisposable
         /// <param name="peerConnection"></param>
         public NativeVideo(IntPtr videoHandle)
         {
-	        _nativeVideoHandle = NativeVideoInterop.Create(videoHandle);
+            _nativeVideoHandle = NativeVideoInterop.Create(videoHandle);
+            if (lookupDictionary.ContainsKey(videoHandle)) lookupDictionary[videoHandle] = this;
+            else lookupDictionary.Add(videoHandle, this);
```

Comment:
This is enough, [if the key doesn't exist it gets created](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.item?view=netstandard-2.0):
```suggestion
            lookupDictionary[videoHandle] = this;
```